Switch to using Puppetlabs apt repo

jblaine opened the pull request:
Tested with "precise" 12.x ships, should adapt to any Ubuntu
on its own. Targets Puppet 2.7.\* as the gem install used to.

jblaine committed:
Use Puppetlabs repo not gems

Tested with "precise" 12.x ships, should adapt to any Ubuntu
on its own. Targets Puppet 2.7.* as the gem install used to.

buildhive commented:
[R. Tyler Croy » blimpy #69](https://buildhive.cloudbees.com/job/rtyler/job/blimpy/69/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

jblaine committed:
Intro .provision_on_start per-ship setting

Defaults to true, of course (provision on start). Setting
ship.provision_on_start (based on the example Blimpfile)
will cause no provisioning at ship start.

jblaine commented:
Sorry, I should have pushed my feature branch up to github and done the PR from that. Hope this is okay. I'm slowly getting my git + GH collaboration chops going.

buildhive commented:
[R. Tyler Croy » blimpy #70](https://buildhive.cloudbees.com/job/rtyler/job/blimpy/70/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

jblaine committed:
Allow CSV output for blimpy status

buildhive commented:
[R. Tyler Croy » blimpy #71](https://buildhive.cloudbees.com/job/rtyler/job/blimpy/71/) SUCCESS
This pull request looks good
[(what's this?)](https://www.cloudbees.com/what-is-buildhive)

rtyler commented:
It looks like there are some conflicts that prevent me from merging this, could I trouble you to resolve them and update the req?

jblaine commented:
So pull from your master and rework this PR? If so, sure. LMK.

rtyler commented:
That should work, probably some really minimal conflicts

rtyler commented:
On Thu, 25 Apr 2013, Jeff Blaine wrote:

> So pull from your master and rework this PR? If so, sure. LMK.

I think so, there shouldn't be too many conflicts
